notebook/services/contents/tests/test_manager.py

```diff
@@ -528,3 +528,34 @@ def test_check_and_sign(self):
         cm.mark_trusted_cells(nb, path)
         cm.check_and_sign(nb, path)
         assert cm.notary.check_signature(nb)
+
+
+class TestContentsManagerNoAtomic(TestCase):
+
+    def setUp(self):
+        self._temp_dir = TemporaryDirectory()
+        self.td = self._temp_dir.name
+        self.contents_manager = FileContentsManager(
+            root_dir = self.td,
+        )
+        self.contents_manager.use_atomic_writing = False
+
+    def tearDown(self):
+        self._temp_dir.cleanup()
+
+    def new_notebook_no_atomic(self):
```

Comment:
I think tests are only run if they have test in the name, like `test_save_notebook_no_atomic`. I might be wrong about that, but double check that the test is actually being executed.